test/src/Test/Main.purs

```diff
@@ -222,15 +234,15 @@ main = void $ runAff throwException (const (pure unit)) $ jumpOutOnError do
           ]
 
   mountConfig1 = ViewConfig
-    { query: "select * from `/smallZips.json`"
+    { query: unsafePartial $ fromRight $ Sql.parseQuery "select * from `/smallZips.json`"
```

Comment:
I think if we switch from string to sql then we manifest that using sql is convenient, right? Why not use it everywhere? 😉 

```purescript
mountConfig = ViewConfig 
  { query: Sql.buildSelect 
      $ (Sql._relations ?~ Sql.TableRelation { alias: Nothing, path: rootDir </> file "smallZips" <.> "json"}) 
      <<< (Sql._projections .~ (pure $ Sql.projection (Sql.splice Nothing)))
  , var: SM.empty
  }
```

Reply:
Switching to `Sql` for me means one cannot construct QuasarF node without valid `Sql`.

Regarding tests I think it's easier and faster to write Sql String then what you have commented here, also I don't  know Sql's AST that well, to rewrite all this queries.